src/user/containers/my_details.js

```diff
@@ -11,9 +12,14 @@ class MyDetails extends React.Component {
     this.props.fetch_member_user()
   }
 
+  update_membership_type_click (e) {
```

Comment:
I don't think this needs to be on the class really and is probably preferable off it